### ISSUE TITLE
compilation of the development version failing

### ISSUE THREAD

**Issue by secanjuraro** [zdebruine/RcppML#31]
Hi Zach, 
First of all, thank you for this great package. I recently started working with it in R (version 4.2.0 ) and I’m having some issues when trying to compile the developement version with the command : devtools::install_github("zdebruine/RcppML"). Could you help me please ?

I’ve tried to install it in two different computers, both with Windows 11 and Rtools installed, and I keep getting the « package installation had non-zero exit status » error . The first warning message says that the package RcppSparse is not available for my R version, which is the latest one and I just can't download it. Do you have any idea where this could be coming from ? 

Best regards, 
Sonia

**Comment by zdebruine** [comment]
Yep, sorry about this.  New release coming soon that removes the `RcppSparse` dependency.  Just do the following for now:

```
library(devtools)
devtools::install_github("zdebruine/RcppSparse")
devtools::install_github("zdebruine/RcppML")
library(RcppML)
```

**Comment by MarcElosua** [comment]
This works for me!

**Comment by zdebruine** [comment]
Fixed with 4cdc6a70cbbab3e32ec43da24151c51d0fd0dbff.  Sparse matrix support will not be integrated into the RcppML package itself.  No additional dependencies.
